src/PeopleSDKAdapter.js

```diff
@@ -184,4 +185,27 @@ export default class PeopleSDKAdapter extends PeopleAdapter {
 
     return this.getPersonObservables[ID];
   }
+
+  /**
+   * Returns an observable that emits people list
+   *
+   * @param {string} query search query
+   * @returns {external:Observable.<Person[]>} Observable that emits person list based on search query
+   */
+  getPeopleList(query) {
+    logger.debug('PEOPLE', query, 'getPeopleList()', ['called with', {query}]);
+
+    return new Observable(async (observer) => {
+      try {
+        const peopleList = await this.datasource.people.list({displayName: query});
+
+        observer.next(peopleList.items);
+        logger.debug('PEOPLE', query, 'getPeopleList()', ['emit persons list', peopleList.items]);
+        observer.complete();
+      } catch (error) {
+        logger.error('PEOPLE', query, 'getPeopleList()', ['error fetching person list'], error);
```

Comment:
the third parameter of logger functions is a message (a string) or an array of strings/objects that gets translated into a string by the formatter.

In case you have a single string, there is no need for square brackets, it's just a message.